Alignment/APEEstimation/plugins/ApeEstimator.cc

```diff
@@ -2107,16 +2105,16 @@ ApeEstimator::isHit2D(const TrackingRecHit &hit) const
       if (detId.subdetId() == PixelSubdetector::PixelBarrel || detId.subdetId() == PixelSubdetector::PixelEndcap) {
         return true; // pixel is always 2D
       } else { // should be SiStrip now
-	const SiStripDetId stripId(detId);
-	if (stripId.stereo()) return true; // stereo modules
+  const SiStripDetId stripId(detId);
+  if (stripId.stereo()) return true; // stereo modules
         else if (dynamic_cast<const SiStripRecHit1D*>(&hit)
-		 || dynamic_cast<const SiStripRecHit2D*>(&hit)) return false; // rphi modules hit
-	//the following two are not used any more since ages... 
+     || dynamic_cast<const SiStripRecHit2D*>(&hit)) return false; // rphi modules hit
+  //the following two are not used any more since ages... 
         else if (dynamic_cast<const SiStripMatchedRecHit2D*>(&hit)) return true; // matched is 2D
         else if (dynamic_cast<const ProjectedSiStripRecHit2D*>(&hit)) {
-	  const ProjectedSiStripRecHit2D* pH = static_cast<const ProjectedSiStripRecHit2D*>(&hit);
-	  return (this->isHit2D(pH->originalHit())); // depends on original...
-	} else {
+    const ProjectedSiStripRecHit2D* pH = static_cast<const ProjectedSiStripRecHit2D*>(&hit);
+    return (this->isHit2D(pH->originalHit())); // depends on original...
+  } else {
```

Comment:
@mteroerd in this file, I see a lot of whitespace-only changes that do not even improve readability. Either drop those changes, or use them to actually align the lines
I would propose to squash all these corrections into a smaller number of commits for a cleaner commit history.

Reply:
Sorry for that, I think my text editor did these changes automatically. I will try to fix the indentation in one commit and then apply the other suggested changes.